README.md

```
@@ -1,264 +1,174 @@
 
----
+## **Available Commands (for Development)** 📜
 
-## **Available Commands** 📜
+Inside the cloned project directory:
 
-- `ng serve`: Start the development server.
-- `ng build`: Build the project for production.
-- `ng test`: Run unit tests.
-- `ng lint`: Lint the codebase for errors.
+  - `ng serve`: Start the development server.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)